view/rooms/app.js

```diff
@@ -408,50 +408,37 @@ function iframeBot(iframe, bot, parsedBot, currentUser) {
   const iframeJS =
   `
   <script>
-    function outputSize(e, browser) {
-      if (browser === 'chrome') {
-        parent.postMessage(
-          {
-            "name": "${bot.name}",
-            "height": e[0].target.scrollHeight
-          }, "*"
-        );
-      } else {
-        parent.postMessage(
+    function outputSize() {
+      const el = document.getElementById("${bot.name}");
+      // Sometimes clientHeight was > scrollHeight so we need to get the max
+      const botHeight =  Math.max(el.clientHeight, el.scrollHeight);
+   
+      parent.postMessage(
         {
           "name": "${bot.name}",
-          "height": document.getElementById("${bot.name}").clientHeight < 100 ?
-            600 :
-            document.getElementById("${bot.name}").clientHeight
+          "height": botHeight < 100 ? 100 : botHeight
```

Comment:
This can written more simply as `Math.max(botHeight, 100)` (and use a named constant for 100)